matlab_simulation/code v1.0/ME597-6-FastSLAM/newCode/ekf_fs.m

```diff
@@ -0,0 +1,28 @@
+function [meanFeatP,covaFeatP,Hmu]=ekf_fs(predFeatPrior,obsFeat,meanFeat,covaFeat,measUncertainty,checkNewFeat)
```

Comment:
Extra descriptions (What this function is used for, what are the inputs and outputs of the function and some other notices might be added) before coding in a function so that your codes could be easily viewed with little preliminary knowledge.